src/django/iow/settings.py

```diff
@@ -266,7 +266,25 @@
             'Project': 'iow',
         },
         'AWS_REGION': 'us-east-1',
-    }
+    },
+    'production': {
+        'TASK_DEFINITION_NAME': 'ProductionAppCLI',
+        'CONTAINER_NAME': 'django',
+        'CLUSTER_NAME': 'ecsProductionCluster',
+        'LAUNCH_TYPE': 'FARGATE',
+        'PLATFORM_VERSION': '1.4.0',
+        'SECURITY_GROUP_TAGS': {
+            'Name': 'sgProductionAppEcsService',
+            'Environment': 'Production',
+            'Project': 'iow',
+        },
+        'SUBNET_TAGS': {
+            'Name': 'PrivateSubnet',
+            'Environment': 'Production',
+            'Project': 'iow',
+        },
+        'AWS_REGION': 'us-east-1',
+    },
```

Comment:
Additional trailing comma, not sure if that matters/affects things

Reply:
Discussed offline, this is needed.  Approved

Reply:
Trailing commas are allowed in Python, and enforced by our linter. Wouldn't work without it.